src/custom_modules/authorization/authMiddleware.ts

```diff
@@ -1,56 +1,50 @@
 import { DAO } from "../DAO";
-import {
-  auth as auth0,
-  requiredScopes,
-  claimCheck,
-  JWTPayload,
-  claimIncludes,
-} from "express-oauth2-jwt-bearer";
-import { authAudience, authRoleClaim, authIssuerBaseURL, authUserIdClaim } from "../../config";
-
-const authorizationRoles = require("../../enums/authorizationRoles.js");
+import { auth as auth0, claimCheck, JWTPayload, claimIncludes } from "express-oauth2-jwt-bearer";
+import { authAudience, authIssuerBaseURL, authUserIdClaim } from "../../config";
+import authorizationPermissions from "../../enums/authorizationPermissions";
 
 // Defaulting to "default" to enable tests to run
 const checkJwt = auth0({
   audience: authAudience || "default",
   issuerBaseURL: authIssuerBaseURL || "default",
 });
 
-const roleClaim = authRoleClaim;
 const userIdClaim = authUserIdClaim;
 
 /**
  * Express middleware, checks if token passed in request grants permission
  * If api is true, stops the request on fail and sends 401 unathorized
  * If api is false, sets req.authorized to true on success and false on fail, then calls next
  * @param {String} permission Shortname permission
- * @param {Boolean} [api=true] Indicates whether the request is an api request or a view request
  */
-export const auth = (permission, api = true) => {
```

Comment:
Unused

Reply:
Yep, it's an old relic from when we had our custom authorization solution :+1: